scripts/generators/typescript.js

```diff
@@ -0,0 +1,150 @@
+"use strict";
+
+const fs = require("fs");
+const t = require("../../packages/babel-types");
+const { stringifyValidator } = require("./utils");
```

Comment:
Since this script should be usable on node 4, can you avoid destructuring?

Reply:
Done